pkg/asset/machines/ovirt/machines.go

```diff
@@ -0,0 +1,70 @@
+// Package ovirt generates Machine objects for ovirt.
+package ovirt
+
+import (
+	"fmt"
+
+	machineapi "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/ovirt"
+
+	ovirtprovider "github.com/ovirt/cluster-api-provider-ovirt/pkg/apis/ovirtclusterproviderconfig/v1alpha1"
```

Comment:
nit: should go in middle block

Reply:
I think I need to set my IDE to stop doing that...